src/towncrier/build.py

```diff
@@ -72,9 +71,9 @@ def _main(
     except Exception as e:
         if isinstance(e, ConfigError):
             print(e)
+            sys.exit(1)
         else:
-            traceback.print_exc(file=sys.stderr)
-        raise
+            raise
```

Comment:
I think that the ConfigError should also be sent to stderr... as it is still an error. Just that is an error that doesn't require a traceback.

Maybe use this to always exit (raise SystemExit) in `_main` ... but maybe your fix work better with Click.... so is ok.
 
```python
    if isinstance(e, ConfigError):
        print(e, file=sys.stderr)
    else:
        traceback.print_exc(file=sys.stderr)
    sys.exit(1)
```

Reply:
Seems I forgot to go back and convert this to `except ConfigError as e:`...  But, to your comments, sure, I'll put the error on `stderr`.  For the traceback, I figured that an uncaught exception ends up on `stderr` anyways.  That was the idea with the bare `raise` (or after adjusting to `except ConfigError`, nothing)  Though, the explicit `sys.exit(1)` does make the exit code more...  explicit.  I don't know if that's good for unexpected exceptions or not.

As to Click, it does offer its own version of this mechanism.  I started to switch to that briefly before seeing that this was already here.  I figured I would leave this focused and consider a full application switch over to Click exceptions separately (if at all).  So, I don't think Click needs to be significantly included in consideration how this should be written.